### PR TITLE
Sending models via gftp

### PR DESCRIPTION
**Pull request by pnowosie**
This PR resolves following JIRA ticket

 
- [APPS-174](https://golemproject.atlassian.net/browse/APPS-174)

### Why
- Before publishing it on mainnet we need to be sure it won’t break.

### What
- Ensure that galatea could be run smoothly on testnet providers (at least 90% successful runs).
- Solve the issue with long VM image download time:

### Assumptions 

[-] _?_

### How to test

[-] _?_

### CODE REVIEW

File: server/api/requirements.txt [hunk 1]
```diff
@@ -2,4 +2,9 @@ Quart==0.14.1
 Quart-CORS==0.4.0
 aiofiles==0.6.0
 aiohttp==3.7
-yapapi==0.5.5
+yapapi==0.6.1
+
```
Comment (Wiezzel) [inline server/api/requirements.txt:6]: Please insert a comment here why these two groups of imports are separated.

File: server/api/src/classifier.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import aiofiles
 import os
 import uuid
+from typing import List, Optional, Set, Tuple
```
Comment (Wiezzel) [inline server/api/src/classifier.py]: Please use `if TYPE_CHECKING` for typing imports.

File: server/api/src/classifier.py [hunk 3]
```diff
@@ -16,9 +17,22 @@
 
 from yapapi.log import enable_default_logger, log_summary, log_event_repr  # noqa
 from yapapi.package import vm
+from yapapi.executor.strategy import (DummyMS, SCORE_REJECTED)
```
Comment (Wiezzel) [inline server/api/src/classifier.py]: Why the parentheses?

File: server/api/src/classifier.py [hunk 4]
```diff
@@ -95,11 +123,12 @@ async def service_start(yagna_app: Yagna) -> None:
             package=package,
             max_workers=1,
             budget=10.0,
-            timeout=timedelta(minutes=30),
+            timeout=timedelta(minutes=99),
             subnet_tag=os.getenv("YAPAPI_SUBNET_TAG"),
             driver=os.getenv("YAPAPI_DRIVER", "zksync"),
             network=os.getenv("YAPAPI_NETWORK", "rinkeby"),
             event_consumer=log_summary(log_event_repr),
+            strategy=DenylistMS(os.getenv("DENY_LIST", "").split())
```
Comment (Wiezzel) [inline server/api/src/classifier.py:134]: Some comment/mention in README about how to use this `DENY_LIST` would be nice.

File: server/yagna/Makefile [hunk 5]
```diff
@@ -1,18 +1,10 @@
...
-FILE := golem-requestor_$(YAGNA_RELEASE)_amd64.deb
-
 all: build
 
-$(FILE):
-	curl -L -o $(FILE) $(RELEASE_URL)/$(YAGNA_RELEASE)/$(FILE)
-
-.PHONY: clean download
-download: $(FILE)
+.PHONY: clean
```
Comment (Wiezzel) [inline server/yagna/Makefile]:
```suggestion
.PHONY: build clean
```

File: server/api/src/classifier.py (hunk 4 above)
Comment (pnowosie) [reply server/api/src/classifier.py:134]: https://github.com/golemfactory/galatea/pull/16/commits/29de452f1c19af2e5d0552582174871a86eda57b#diff-52af042a8f7f47fd511d97c15ec95d7b89eaa93428af8cc3227bc1112553f728

File: server/api/README.md [hunk 6]
```diff
@@ -7,4 +7,28 @@ To remove docker artifacts run `make clean`
...
+### Run classifier from api service
+```bash
+docker-compose run \
+  -e YAGNA_APPKEY="$(jq -r '.values[0][1]' yagna_data/app_key)" \
+  -e YAPAPI_SUBNET_TAG=devnet-beta.2 \
+  -e DENY_LIST="" \
+  api python classifier.py
+```
+
+Now, you are able to exclude some providers providing space delimited list of provider names for `DENY_LIST` env var.
```
Comment (Wiezzel) [inline server/api/README.md:32]: Let's wait until someone runs a crippled node with a space in its name. :stuck_out_tongue:

**Review by Wiezzel — APPROVED** [review]

**Pull request merged**